crates/agglayer-aggregator-notifier/src/certifier/mod.rs

```diff
@@ -12,7 +11,9 @@ use agglayer_prover_types::{
     },
 };
 use agglayer_storage::stores::{PendingCertificateReader, PendingCertificateWriter};
-use agglayer_types::{Certificate, Height, LocalNetworkStateData, NetworkId, Proof};
+use agglayer_types::{
+    primitives::Address, Certificate, Height, LocalNetworkStateData, NetworkId, Proof,
+};
```

Comment:
Any opinions about having some commonly used primitives (such as `H256`, `Address`) also re-exported at the top-level of `agglayer_types`?

Reply:
Not sure - happy to hear other opinions.

Reply:
I'm ok with that